chennai.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+header: Reaching the Venues
+title: Reaching the Venues
+bloglike: true
+---
+
+This page is to help the attendees reach Chennai and the conference venues.
+
+<br>
+
+### Getting to PyCon India 2019
+
+**By Plane**
+
+Chennai has an international airport that is well connected to Europe, the Middle East, US,
+and South East Asia. It is also well connected by domestic flights.
+
+**Chennai Trade Center(Conference)**
+
+The easiest option is to take a metro from Chennai Airport towards Washermenpet Metro and alight
+at Guindy Metro. From the metro station, you can board one of these buses - 49, 54, 54D and
+get down at Contonment Cemetery. The venue is 200 meters from here. Alternatively, you can hail
+a cab from ride sharing services like Uber or Ola. Please **do not** try to get an autorickshaw
+as they usually rip off outsiders. Same goes for rental cabs at the airport as well.
+
+**IITM Research Park(Workshops and Dev Sprints)**
+
+Again, the easiest option is to take a metro towards Washermenpet and alight at Saidapet Metro.
+From there, a bus towards Besant Nagar should take you to IIT Madras. The bus routes are
+5E, 23C and 47. The IIT-M bus stop might also be referred to CLRI.
```

Comment:
IITMRP has a different entrance from IITM. It is just behind the Ramanujan IT Park.